docs/src/runtime/programs.md

```diff
@@ -5,17 +5,17 @@ sidebar_label: Native Programs
 ---
 
 Solana contains a small handful of native programs, which are required to run
-validator nodes. Unlike third-party programs, the native programs are part of
+validator nodes. Unlike the third-party programs, the native programs are part of
```

Comment:
I don't think the grammar was wrong here. In fact, this actually makes less sense because the "the" makes the reader think there are special third-party programs being referred to here.

This is almost pedantic, but I think maybe drop this change please.